ovn_cluster.sh

```diff
@@ -8,10 +8,10 @@ set -o errexit
 RUNC_CMD="${RUNC_CMD:-docker}"
 
 BASE_IMAGE="ovn/cinc"
```

Comment:
I know it's a bit unrelated but could you also remove BASE_IMAGE now that you're touching the IMAGE variables.  I think we never used it anywhere.

Reply:
Ack.  Done